docs/user_guide/4_pipelines.md

```diff
@@ -0,0 +1,14 @@
+## Vision Pipelines
+
+Pipelines can simplify complex vision tasks by breaking it into a sequence of operations that are applied to every image. Images are modeled as `landingai.vision_pipeline.Frame` and sequences of images as `landingai.vision_pipeline.FrameSet`. Most image sources will produce a `FrameSet` even when it contains a single image (e.g. `landingai.vision_pipeline.NetworkedCamera`). This is convenient as it allows subsequent stages of the pipelines to introduce new derived frames as part of the processing (e.g. when extracting faces from an initial Frame into new Frames).
```

Comment:
grammar/typo: "sequences of images as `landingai.vision_pipeline.FrameSet`" -> "sequences of images is modeled as `landingai.vision_pipeline.FrameSet`"

It'd be helpful to expand on this topic: "Most image sources will produce a `FrameSet` even when it contains a single image" 
I think some additional context to share would be: `NetworkedCamera` is an iterator of captured frame. Each frame/image is modeled as an `FrameSet` (not a Frame), though there is only one image in the `FrameSet`.
Adding an example is even better.